tests/unit/test_openapi/test_responses.py

```diff
@@ -42,6 +41,8 @@
 from tests.models import DataclassPerson, DataclassPersonFactory
 from tests.unit.test_openapi.utils import PetException
 
+T = TypeVar("T")
+
```

Comment:
Any particular reason to prefer local definition over the import?

Reply:
Two things: `T` is not declared in `litestar.response.base.__all__`, and it seems (a little) weird to me to import an unbounded type variable from a sibling module. I'm not sure what the project convention is though, so happy to revert this if you think it's inappropriate

Reply:
Both of those are valid reasons.

We do often import names that aren't declared in `__all__` into tests b/c we often don't include names in there that we don't want in the public interface yet still want to test.

On point two, if I was writing this module today I wouldn't have imported `T` either. Again, no strict convention, but I would say that the common practice is to not import these.

As a counter argument, minor changes do have a cost in that someone has to review them. So as long as you balance that fact against the changes you propose then I'm happy enough.